### ISSUE TITLE
Sequence Memory/Snippets location?

### ISSUE THREAD

**Issue by Audio-Rochey** [PatternAgents/Haptic_DA7280#2]
Tom, it looks like you're downloading the entire snipped memory in the reset condition in Haptic_DA7280.cpp - right?
Is your expectation that folks would take the exported memory map from the DA7280 GUI and import it into this array of values?

**Comment by moxondesign** [comment]
Yes. I expect that most users would import that table from the DA7280 GUI configuration output file.
The library is just a simple Arduino I/O interface for loading/saving those settings.

**Comment by KianYeo** [comment]
Hi, I am trying to load my own exported memory map exported from the DA7280 GUI. However I am confused how to update the array of values in _const struct scr_type haptic_reset[]_
Do I replaced all the array pairs starting from {0x03, 0x00} all the way till the end? The haptic memory is only 100 bytes but I think there are more than 100 pairs of values. Moreover, the exported values are something like this: 3, 3, 9, 10, 12, 13, 14, 15, 95, 32, 255, 255, 240, 25, 26, 19, ....

Wonder if you can further advise on this. Thanks in advance!
